homeassistant/components/device_tracker/geofancy.py

```diff
@@ -24,6 +26,9 @@ def setup_scanner(hass, config, see):
     global _SEE
     _SEE = see
 
+    global _HASS
```

Comment:
Please don't do this. You can get access to the hass object using `handler.server.hass`